interface/patient_tracker/patient_tracker.php

```diff
@@ -357,10 +357,10 @@
         <div class="col-md-12">
             <div class="text-center row mx-auto divTable">
                 <div class="col-sm-12" id="loader">
-                    <div class="text-center">
-                        <i class="fa fa-spinner fa-pulse fa-fw" style="font-size: 140px; color: var(--gray700); padding: 20px"></i>
-                        <h2><?php echo xlt('Loading data'); ?>...</h2>
+                    <div class="spinner-border" role="status">
+                        <span class="sr-only">Loading...</span>
```

Comment:
Your gonna need to use the xlt class to allow for OpenEMR translation. Replace the loading text with this:
`<?php echo xlt('Loading data'); ?>...`